h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2339,6 +2359,41 @@ else if (gaussian.equals(_parms._family) && Link.identity.equals(_parms._link))
       }
     }
 
+    /***
+     * Estimate dispersion factor using maximum likelihood.  I followed section IV of the doc in 
+     * https://h2oai.atlassian.net/browse/PUBDEV-8683 . 
+     */
+    public double estimateMLSE(GLMTask.ComputeMLSETsk mlCT, double alpha, double[] beta) {
+      double constantValue = mlCT._wsum + mlCT._sumlnyiOui - mlCT._sumyiOverui;
+      DataInfo dinfo = _state.activeData();
+      Frame adaptedF = dinfo._adaptedFrame;
+      int count = 0;
+      long currTime = System.currentTimeMillis();
+      long modelBuiltTime = currTime - _model._output._start_time;
+      while (true) {
```

Comment:
I hard a hard time figuring out the stopping criteria here - I think this should really be a for-loop 0..100

Reply:
I don't really know how many iterations to use here.  So using a for loop can be risky.  If it is too short, won't get good result.  If too long, waste time.

Reply:
got it, thanks for explaining